monopoly/src/main/java/gameset/functionality/Game.java

```diff
@@ -130,13 +140,15 @@ private void viewProperty(Property property, Player player) throws IOException {
                 purchaseProperty(property, player);
 
             }
-        } else {
+        } else if (!Objects.equals(property.getOwner(), player)) {
```

Comment:
Does this work as intended right now? I'd expect (given that owner is a String at this moment) that this should be comparing the names of the player:
```suggestion
        } else if (!player.getNameNoColor().equalsIsIgnoreCase(property.getOwner()) {
```